pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -351,6 +360,14 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 		c.recorder.Event(etcdstorage, corev1.EventTypeWarning, ResourceReclaimed, msg)
 	}
 
+	// Once EtcdStorage is successfully deployed, deploy the certificates in the API server namespace.
+	if err = c.updateAPIServerServingCAConfigMap(etcdstorage); err != nil {
```

Comment:
if this fails, we still want to update the secrets.  We want to complete as much work as possible because some clients may still be valid.